VogtPayroll4/EmployeePayrollConsoleReader.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace VogtPayroll4
+{
+    class EmployeePayrollConsoleReader
+    {
+        public List<Employee> CreateNewEmployee()
+        {
+
+            StreamReader file = new StreamReader(@"C:\Users\Gabriel\source\repos\VogtPayroll4\MyTest.txt");
+            string line;
+            List<Employee> employees = new List<Employee>();
+
+            while ((line = file.ReadLine()) != null)
+            {
+                string[] employeeProperties = line.Split(',');
+                Employee emp = new Employee(employeeProperties[0], int.Parse(employeeProperties[1]), int.Parse(employeeProperties[2]), int.Parse(employeeProperties[3]));
```

Comment:
Can reduce from 2 lines to 1:
```c#
employees.add(new Employee(employeeProperties[0], int.Parse(employeeProperties[1]), int.Parse(employeeProperties[2]), int.Parse(employeeProperties[3])));
```

It seems minor, but this is the type of feedback I'd give to other junior programmers on my team. 

Another way to look at it.. there's no reason to assign the `emp` variable if all you're going to do is add it to a list.